pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: postgres
 description: PostgreSQL database driver. Supports statement reuse and binary protocol.
-version: 2.6.1
+version: 2.8.1
```

Comment:
The next semantic version number for something with a new feature would be 2.7.0

Reply:
fix